assets/js/blocks/cart-checkout/cart/full-cart/style.scss

```diff
@@ -63,13 +63,37 @@ table.wc-block-cart-items {
 
 .wc-block-cart-item__total {
 	vertical-align: bottom;
+	font-size: 16px;
+	line-height: 19px;
 }
 
 .wc-block-cart-item__full-price {
-	color: $core-grey-light-800;
+	// inline on mobile, so line prices are aligned vertically
+	display: inline-block;
+	margin-right: 0.5em;
+
+	color: $core-grey-dark-400;
 	text-decoration: line-through;
 }
 
+.wc-block-cart-item__line-total {
+	color: $black;
+}
+
+.wc-block-cart-item__discount-badge {
+	// hidden on mobile
+	display: none;
+
+	span {
+		background-color: $core-grey-dark-600;
+		border-radius: 3px;
+		color: $white;
+		font-size: 12px;
+		padding: 0 1em;
+		text-transform: uppercase;
```

Comment:
And this inside the other one `@include breakpoint( ">782px" ) {`?

Reply:
For now I'm leaving these as common styles - the element is hidden on mobile, rather than styled differently. This ensures that if it's displayed on mobile (e.g. merchant / theme overrides, or we change mobile layout or add an option in future) the "badge" styles are available.

E.g. 
<img width="368" alt="Screen Shot 2020-01-14 at 11 22 05 AM" src="https://user-images.githubusercontent.com/4167300/72297139-22948180-36c0-11ea-9055-2a6c2d0c287b.png">